### DISCUSSION TITLE
Add ktlint-gradle with Springboot using Kotlin

### DISCUSSION THREAD

**Discussion by Debanshu777** [JLLeitschuh/ktlint-gradle#535] [Q&A]
Hi, 
I add ktlint to a personal SpringBoot project which used Kotlin and Gradle. It worked ideally but I want to ask is it workable enough for pre-production checks?

![Screenshot (122)](https://user-images.githubusercontent.com/50791485/133687824-fe989cb1-1ea3-4019-a76b-c1bc3c41cf61.png)
![Screenshot (123)](https://user-images.githubusercontent.com/50791485/133687830-bfd49638-60c6-4116-ae39-fee556e94ab2.png)
?

**Comment by JLLeitschuh** [comment]
> is it workable enough for pre-production checks?

Can you clarify what you're asking here?
